gapis/api/vulkan/resources.go

```diff
@@ -1049,6 +1071,21 @@ func InputAssembly() *api.Stage {
 
 		&api.DataGroup{
 			GroupName: "Input Assembly State",
+			Data: &api.DataGroup_KeyValues{
+				&api.KeyValuePairList{
+					KeyValues: []*api.KeyValuePair{
```

Comment:
This is quite verbose, I would prefer to extract the repeat KeyValuePair{ Name: Value: } into a separate method, or these will start getting out of hand.